l1-contracts/test/foundry/l1/unit/concrete/state-transition/ChainTypeManager/ChainTypeManagerSetters.t.sol

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import {ChainTypeManagerTest} from "./_ChainTypeManager_Shared.t.sol";
+import {GettersFacet} from "contracts/state-transition/chain-deps/facets/Getters.sol";
+import {UtilsFacet} from "foundry-test/l1/unit/concrete/Utils/UtilsFacet.sol";
+import {FeeParams, PubdataPricingMode} from "contracts/state-transition/chain-deps/ZKChainStorage.sol";
+import {Bridgehub} from "contracts/bridgehub/Bridgehub.sol";
+
+import {console} from "forge-std/console.sol";
+
+contract ChainTypeManagerSetters is ChainTypeManagerTest {
+    function setUp() public {
+        deploy();
+    }
+
+    // setPriorityTxMaxGasLimit
+    function test_SuccessfulSetPriorityTxMaxGasLimit() public {
+        address chainAddress = createNewChain(getDiamondCutData(diamondInit));
+        GettersFacet gettersFacet = GettersFacet(chainAddress);
```

Comment:
this thing coupled with `vm.mockCall(address(bridgehub), abi.encodeCall(Bridgehub.getZKChain, chainId), abi.encode(chainAddress));` is repeated in a lot of places, could be united into one function to reduce code repetition

Reply:
Done.